ecms_base/features/custom/ecms_event/config/install/pathauto.pattern.events.yml

```diff
@@ -2,20 +2,31 @@ langcode: en
 status: true
 dependencies:
   module:
+    - language
     - node
 id: events
 label: Events
 type: 'canonical_entities:node'
 pattern: 'events/[node:title]'
 selection_criteria:
-  683b1f5f-b16e-4486-a1a4-766a8f95b390:
+  2b91bbb8-ae1d-4944-98bf-d6b628a417f1:
```

Comment:
Are these UUID's taken from the Covid site? What happens on a different site? Will the changes cause issues on another site?

I notice these changes in all of the content types.

Reply:
I saw that update too - and no, these were exported as features from the local development site. I looked at the rihavens site, and it had the "old" uuid too (for the page pattern), i.e. "4532149e-106e-4e15-b944-14a7007a3011"

https://rihavens.riecms.acsitefactory.com/admin/config/development/configuration/single/export

I'm wondering if these are simply arbitrary, based on some combination of the settings? I would say let's just test them out on stage after the next deploy to ensure things are functional.